pkg/controller/build/buildrequest/assets/Containerfile.on-cluster-build-template

```diff
@@ -28,6 +18,25 @@ COPY --from=extract /etc/machine-config-daemon/currentconfig /etc/machine-config
 # since it should be set by the container runtime / builder.
 RUN container="oci" exec -a ignition-apply /usr/lib/dracut/modules.d/30ignition/ignition --ignore-unsupported <(cat /etc/machine-config-daemon/currentconfig | jq '.spec.config') && \
 	ostree container commit
+{{if and .ExtensionsImage .Extensions}}
+COPY --from={{.ExtensionsImage}} / /run/mco-extensions/os-extensions-content
```

Comment:
question: I wonder if we can mount this instead of copying it? The reason is because copying would increase the final image size with unnecessary content whereas if it is mounted at build-time, the unnecessary content disappears when the image build is finished.

Note: Figuring that out is nice-to-have, not a need-to-have since we can make that adjustment later.

Reply:
Fixed